app/views/layouts/_breadcrumbs.erb

```diff
@@ -1,7 +1,9 @@
 <div class="wrap-outer-breadcrumb layout-band">
   <div class="wrap-breadcrumb" role="navigation" aria-label="breadcrumbs">
     <div class="breadcrumbs">
-      <a href="https://libraries.mit.edu" title="MIT Libraries">Home</a>  &raquo;  <a href="/">Thesis Submission</a>
+      <%= link_to('Home', root_path) %>
```

Comment:
Should the filename be `_breadcrumbs.html.erb`?

Reply:
yes. I "fixed" it from `_breadcrumbs.html` wrongly. Sigh.

Reply:
will fix

Reply:
fixed